improver_tests/standardise/test_StandardiseMetadata.py

```diff
@@ -292,6 +292,16 @@ def test_air_temperature_status_flag_coord_without_realization(self):
         self.assertArrayEqual(result.data, target.data)
         self.assertEqual(result.coords(), target.coords())
 
+    def test_long_name_removed(self):
+        cube = self.cube.copy()
+        cube.long_name = "kittens"
+        print("long_name before test:" + cube.long_name)
+        result = StandardiseMetadata()._remove_long_name_if_standard_name(cube)
+        result.long_name is None, "long_name attribute removed or never added"
+
+        print(result)
```

Comment:
Further print statements to remove in here and the assert statements to be put back.